resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -449,7 +449,6 @@ notice.message.jetbrains.minimum.version=Support for version {0} of {1} is being
 notice.suppress=Don't show this again
 notice.title.jetbrains.minimum.version=AWS Toolkit Deprecation Notice
 resource_selector.create=Create
-s3.bucket.arn.label=Bucket ARN:
 s3.bucket.creation_date.label=Creation Date:
```

Comment:
This can be deleted

Reply:
Bucket name is still used but creation date can be deleted, updated